pom.xml

```diff
@@ -426,7 +426,7 @@
 
 					<!-- Third argument prevents Java from popping up lots of windows on
 						MacOS -->
```

Comment:
I can work on either fixing the tests or fixing these newly introduced tests. Either ways supporting "older laptops" with presumably very little memory sounds infeasible. Even apache spark recommends more than 2GB of memory on the driver.